lib/puppet/parser/functions/simp_opt.rb

```diff
@@ -0,0 +1,49 @@
+module Puppet::Parser::Functions
+  newfunction(:simp_opt, :type => :rkey, :arity => 1, :doc => <<-EOS
```

Comment:
In Puppet 4, I think we're supposed to use [Puppet::Functions.create_function](https://github.com/puppetlabs/puppet-specifications/blob/master/language/func-api.md#creating-a-function)